app/src/main/java/com/kickstarter/viewmodels/FrequentlyAskedQuestionsViewHolderViewModel.kt

```diff
@@ -25,31 +25,39 @@ interface FrequentlyAskedQuestionsViewHolderViewModel {
         fun updatedDate(): Observable<String>
     }
 
-    class ViewModel(@NonNull val environment: Environment) :
-        ActivityViewModel<FrequentlyAskedQuestionsViewHolder>(environment), Inputs, Outputs {
+    class FrequentlyAskedQuestionsViewHolderViewModel() : ViewModel(), Inputs, Outputs {
```

Comment:
This inheritance works only for ViewModels on `Activity/Fragments`. We do not build this viewModel using a factory, and if take a look into the `ViewModel()` constructor source code should never be used without it. 
<img width="711" alt="Screenshot 2023-06-29 at 2 07 14 PM" src="https://github.com/kickstarter/android-oss/assets/4083656/6f5f678d-d2b3-4d9f-8bc6-30d56ea61e45">

Remove this inheritance, instead create a new input method, that will be called from the `onDestroy()` on the viewHolder. There is an example in 
<img width="405" alt="Screenshot 2023-06-29 at 2 11 10 PM" src="https://github.com/kickstarter/android-oss/assets/4083656/81f4c541-8893-44ec-9fda-3bd62bb07ea5">
`BackingAddOnViewHolder`, `BackingAddOnViewHolderViewModel`